linkers/dld

```diff
@@ -7,7 +7,7 @@ LINKER=${LINKER#"d-"}
 ARGS=$@
 
 while (( "$#" )); do
-	if [[ $1 == *"-Wl,--version-script="* ]]; then
+	if [[ $1 == *"-Wl,--version-script="* && ! $ARGS =~ -o.*librustls_platform_verifier.* ]]; then
```

Comment:
This is to make sure that what is happening in the body of 'if' is only executed for object files from libtelio, and not for those from librustls_platform_verifier.